### ISSUE TITLE
More dynamic for table declaration

### ISSUE THREAD

**Issue by gleb619** [eXsio/querydsl-entityql#4]
Entityql could provide more dynamic way to defining entities(tables). In my specific case i doesn't know anything about database at compile time, so i can get it only at runtime. I've slightly modified the code for my needs. Can you review [PR](https://github.com/eXsio/querydsl-entityql/pull/3) and give me feedback.

**Comment by eXsio** [comment]
Hello

Thank you for the PR. Review is ready.

**Comment by gleb619** [comment]
Pr is merged, closing issue

**Comment by eXsio** [comment]
Hi @gleb619. Thank you for your contribution. I am really curious about your specific use case. Would you consider giving a real life usage example and adding a few cents to the README, so that all users may know exactly when and how to use the new code?

**Comment by gleb619** [comment]
Ok. Long story short, we need simplified phpMyAdmin in Java for our business cases. 

Long story long: Once we realized for ourselves that every month we spend several days creating/updating/supporting dictionaries in our unrelated projects. We thought that with a separate module (or a macro service) we could improve the situation. After asking a little bit and searching on the Internet how to solve this problem, we found the [MDM](https://en.wikipedia.org/wiki/Master_data_management) concept. Then we started working on it, persistence layer is obviously a major part of this program. Surfing [the habr](https://habr.com/ru/company/jugru/blog/482218/), i came across this project, and I finally figured out where to start.
